src/main/java/pl/tomaszdziurko/jvm_bloggers/view/admin/AbstractAdminPage.java

```diff
@@ -5,11 +5,17 @@
 import org.apache.wicket.markup.html.link.BookmarkablePageLink;
 import org.apache.wicket.request.mapper.parameter.PageParameters;
 
+import pl.tomaszdziurko.jvm_bloggers.view.admin.blogs.BlogsPage;
 import pl.tomaszdziurko.jvm_bloggers.view.admin.mailing.MailingPage;
 import pl.tomaszdziurko.jvm_bloggers.view.admin.moderation.ModerationPage;
 
+import java.time.format.DateTimeFormatter;
+
 public abstract class AbstractAdminPage extends WebPage {
 
+    public static final DateTimeFormatter DATE_FORMATTER =
```

Comment:
I've seen a similar `DateTimeFormatter` somewhere in the code. Maybe we could move it to `DateTimeUtilities`?